### PR TITLE
docs(badge): badge docs cleanup

### PR DESCRIPTION
**Pull request by nikkimk**
## What I did

1. Updated badge docs #923.

## Testing Instructions

1. View [DP badge docs](https://deploy-preview-966--red-hat-design-system.netlify.app/elements/badge/).
2. Compare to [Badge doc](https://docs.google.com/document/d/1arJJbhGj3nLxWRzCVH67dhJnpkzHAjgGk57a764YTx0/edit#heading=h.aixz6587nrdt) and to [Docs cleanup checklist](https://github.com/RedHat-UX/red-hat-design-system/wiki/Docs-Cleanup-Q2-2023-Checklist).

## Notes to Reviewers
The following are WIP dependent on other issues/PRs:
- [Image width styling](https://github.com/RedHat-UX/red-hat-design-system/pull/891)
- [Code tab styling](https://github.com/RedHat-UX/red-hat-design-system/issues/969)

### CODE REVIEW

File: .changeset/red-melons-begin.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+Added a summary to `<rh-badge>`
```
Comment (bennypowers) [inline .changeset/red-melons-begin.md:5]:
```suggestion
`<rh-badge>`: corrected and updated custom elements manifest
```

File: elements/rh-badge/rh-badge.ts
Comment (bennypowers) [inline elements/rh-badge/rh-badge.ts:1]: can we add in `@cssprop`s here, too?

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: elements/rh-badge/rh-badge.ts [hunk 2]
```diff
@@ -17,7 +30,6 @@ export class RhBadge extends BaseBadge {
 
   /**
    * Denotes the state-of-affairs this badge represents
```
Comment (bennypowers) [inline elements/rh-badge/rh-badge.ts:32]: we should specify the semantic meaning of each option here
```suggestion
   * Denotes the state-of-affairs this badge represents
   * - **info**: yadda yadda
   * - **success**: etc etc
```

Need @coreyvickery's design input on that

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: elements/rh-badge/docs/20-guidelines.md [hunk 3]
```diff
@@ -1,13 +1,11 @@
 ## Usage 
 Use a badge to reflect counts like number of objects, events, or unread items.
 ## Badge vs. tag 
-If you need to add specific text captions to elements, consider using a Tag instead.
+If you need to add specific text captions to elements, consider using a <a href="/elements/tag">Tag</a> instead.
```
Comment (bennypowers) [inline elements/rh-badge/docs/20-guidelines.md:4]:
```suggestion
If you need to add specific text captions to elements, consider using a [Tag](/elements/tag/) instead.
```

File: elements/rh-badge/docs/20-guidelines.md [hunk 4]
```diff
@@ -1,13 +1,11 @@
...
 ## Counter number 
 The width of a badge varies based on the counter number. Using a number larger than the threshold will display a + at the end. For example, if <code>999</code> is the threshold, using <code>1,000</code> or larger will display <code>999+</code>.
```
Comment (bennypowers) [inline elements/rh-badge/docs/20-guidelines.md:6]:
```suggestion
The width of a badge varies based on the counter number. Using a number larger than the threshold will display a + at the end. For example, if `999` is the threshold, using `1,000` or larger will display `999+`.
```

prefer markdown syntax

File: elements/rh-badge/docs/10-style.md [hunk 5]
```diff
@@ -29,10 +29,10 @@ All badges have the same height and border radius.
           alt="Badge spacing and minimum width",
           src="../badge-space-and-width.png" %}
 
-| Property | Current value |
-| -------- | -------- |
-| ![8 pixel spacer]({{ '../badge-8px-spacer.png' | url }}){width=8} | `8px 0.5rem` |
-| Minimum width | `32px 2.0rem` |
+| Property                                                          | Current value |
+| -------- ---------------------------------------------------------| ------------- |
+| ![8 pixel spacer]({{ '../badge-8px-spacer.png' | url }}){width=8} | `8px 0.5rem`  |
```
Comment (bennypowers) [inline elements/rh-badge/docs/10-style.md:34]: must #949

**Comment by nikkimk** [comment]
@bennypowers PTAL
@coreyvickery Also TAL

**Comment by coreyvickery** [comment]
@nikkimk Looks fine.

**Review by coreyvickery — APPROVED** [review]
Looks good.

File: docs/_includes/accessibility/1.4.1-A.md [hunk 6]
```diff
@@ -0,0 +1 @@
+- [SC 1.4.1 Ise of Colot](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) (Level A)
```
Comment (bennypowers) [inline docs/_includes/accessibility/1.4.1-A.md]:
```suggestion
- [SC 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) (Level A)
```

File: elements/rh-badge/docs/20-guidelines.md [hunk 7]
```diff
@@ -1,16 +1,42 @@
 ## Usage 
 Use a badge to reflect counts like number of objects, events, or unread items.
-## Badge vs. tag 
+
+## Variations
+A badge includes various status colors that communicate different semantic meanings.
+
+{% alert title="Warning" %}
```
Comment (bennypowers) [inline elements/rh-badge/docs/20-guidelines.md:7]:
```suggestion
{% alert title="Warning", state="warning" %}
```

**Review by bennypowers — CHANGES_REQUESTED** [review]

**Review by bennypowers — APPROVED** [review]

**Pull request merged**